### DISCUSSION TITLE
New connection per second?

### DISCUSSION THREAD

**Discussion by QuadTriangle** [uNetworking/uWebSockets#1612] [Q&A] [answered]
I have done a pretty limited testing. My connection rates is not very high.

What is the expected new WebSocket connection per second in both node.js and cpp? I only need an approximate number as a reference. Do you have any advice if I am optimizing only for new connection rates for unique users? My use cases is to find online users count.

**Comment by uNetworkingAB** [comment, marked as answer]
With SSL it takes a lot of time, 200 per second maybe. Without, many thousands
